python/src/otel/otel_sdk/requirements-nodeps.txt

```diff
@@ -27,4 +27,3 @@ opentelemetry-instrumentation-sqlite3==0.42b0
 opentelemetry-instrumentation-starlette==0.42b0
 opentelemetry-instrumentation-tornado==0.42b0
 opentelemetry-instrumentation-wsgi==0.42b0
-
```

Comment:
What about the `0.42b0` portion of the version... does that matter?  What is the difference?

Reply:
I'll use only `1.21.0` as the version of the layer, we also use `0.42b0` for some packages that are not stable yet, but `1.21.0` is a sufficient id for a release.